fix(cmd)_: nodekey as env var in pn-server

mendelskiv93 opened the pull request:
Ref.: https://github.com/status-im/infra-push-notify/issues/11

mendelskiv93 committed:
fix(cmd)_: nodekey as env var in pn-server

Ref.: https://github.com/status-im/infra-push-notify/issues/11

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (27)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6738/1/) | 2025-07-05 08:31:10 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250705-082821-fbcd84-pr6738.zip) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6738/1/) | 2025-07-05 08:31:18 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250705-082824-fbcd84-pr6738.zip) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6738/1/) | 2025-07-05 08:31:28 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250705-082819-fbcd84-pr6738.aar) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6738/1/) | 2025-07-05 08:33:24 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250705-082826-fbcd84-pr6738.zip) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/1/) | 2025-07-05 08:33:44 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/1//consoleText) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6738/1/) | 2025-07-05 08:34:24 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250705-082831-fbcd84-pr6738.zip) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6738/1/) | 2025-07-05 08:37:22 | ~9 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250705-082820-fbcd84-pr6738.zip) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6738/1/) | 2025-07-05 08:37:55 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250705-082822-fbcd84-pr6738.zip) |
| :heavy_check_mark: | fbcd84d5 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/1/) | 2025-07-05 08:53:23 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/1//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/2/) | 2025-07-07 13:02:35 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/2//consoleText) |
| :heavy_check_mark: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6738/2/) | 2025-07-07 13:03:25 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250707-130030-0ba67e-pr6738.zip) |
| :heavy_check_mark: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6738/2/) | 2025-07-07 13:03:28 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250707-130023-0ba67e-pr6738.aar) |
| :heavy_check_mark: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6738/2/) | 2025-07-07 13:03:33 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250707-130026-0ba67e-pr6738.zip) |
| :heavy_check_mark: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6738/2/) | 2025-07-07 13:05:31 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250707-130033-0ba67e-pr6738.zip) |
| :heavy_check_mark: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6738/2/) | 2025-07-07 13:06:35 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250707-130038-0ba67e-pr6738.zip) |
| :heavy_multiplication_x: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/2/) | 2025-07-07 13:06:49 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/2//consoleText) |
| :heavy_check_mark: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6738/2/) | 2025-07-07 13:07:17 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250707-130025-0ba67e-pr6738.zip) |
| :heavy_check_mark: | 0ba67e4f | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6738/2/) | 2025-07-07 13:10:27 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250707-130027-0ba67e-pr6738.zip) |
| | | | | | | |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6738/3/) | 2025-07-07 13:05:46 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250707-130334-299787-pr6738.aar) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6738/3/) | 2025-07-07 13:06:54 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250707-130342-299787-pr6738.zip) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6738/3/) | 2025-07-07 13:07:01 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250707-130330-299787-pr6738.zip) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6738/3/) | 2025-07-07 13:10:35 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250707-130536-299787-pr6738.zip) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6738/3/) | 2025-07-07 13:12:33 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250707-130637-299787-pr6738.zip) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/3/) | 2025-07-07 13:12:38 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/3//consoleText) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6738/3/) | 2025-07-07 13:12:46 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250707-130726-299787-pr6738.zip) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6738/3/) | 2025-07-07 13:18:06 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250707-131031-299787-pr6738.zip) |
| :heavy_check_mark: | 29978760 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/3/) | 2025-07-07 13:28:09 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/3//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6738/4/) | 2025-07-08 08:44:49 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250708-084235-2cf7b5-pr6738.aar) |
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6738/4/) | 2025-07-08 08:45:26 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250708-084242-2cf7b5-pr6738.zip) |
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6738/4/) | 2025-07-08 08:45:32 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250708-084238-2cf7b5-pr6738.zip) |
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6738/4/) | 2025-07-08 08:47:15 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250708-084237-2cf7b5-pr6738.zip) |
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/4/) | 2025-07-08 08:47:31 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/4//consoleText) |
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6738/4/) | 2025-07-08 08:47:43 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250708-084245-2cf7b5-pr6738.zip) |
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6738/4/) | 2025-07-08 08:52:19 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250708-084240-2cf7b5-pr6738.zip) |
| :heavy_check_mark: | 2cf7b515 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/4/) | 2025-07-08 09:07:30 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6738/5/) | 2025-07-08 09:19:36 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250708-091718-d3b59c-pr6738.aar) |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6738/5/) | 2025-07-08 09:20:06 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250708-091725-d3b59c-pr6738.zip) |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6738/5/) | 2025-07-08 09:20:14 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250708-091721-d3b59c-pr6738.zip) |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6738/5/) | 2025-07-08 09:21:40 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250708-091719-d3b59c-pr6738.zip) |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6738/5/) | 2025-07-08 09:22:24 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250708-091728-d3b59c-pr6738.zip) |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/5/) | 2025-07-08 09:22:28 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6738/5//consoleText) |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6738/5/) | 2025-07-08 09:25:10 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250708-091722-d3b59c-pr6738.zip) |
| :heavy_check_mark: | d3b59c60 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/5/) | 2025-07-08 09:42:09 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6738/5//consoleText) |

mendelskiv93 committed:
feat(cmd)_: add configurable TCP and UDP ports for `push-notification-server`

Random port assignment makes infrastructure monitoring and service
discovery difficult. Fixed ports enable proper health checks,
port mapping, and integration with existing monitoring systems.

- https://github.com/status-im/infra-push-notify/issues/11

jakubgs commented on cmd/push-notification-server/main.go:
```diff
@@ -43,7 +43,8 @@ var (
 	wakuFleet       = flag.String("waku-fleet", "status.prod", "Waku fleet to use")
 	wakuFleetConfig = flag.String("waku-fleet-config", "", "path to Waku fleet config file")
-
+	Port            = flag.Int("tcp-port", 0, "Libp2p TCP port")
+	UDPPort         = flag.Int("udp-port", 0, "Libp2p UDP port")
```
Why is the default `0` which is an invalid value instead of something standard like `30303`?

mendelskiv93 replied:
I believe `0` requests random OS-assigned port and I left it for backward compatibility, though a sensible default like `30303` would indeed be better.
